mappings/net/minecraft/item/FlintAndSteelItem.mapping

```diff
@@ -1 +1,7 @@
 CLASS bbq net/minecraft/item/FlintAndSteelItem
+	METHOD a isLightableBlock (Lbvj;)Z
```

Comment:
`hasLitState`?

Reply:
More true as to what the method is used for in practice. "canBeLit"?

Reply:
Hmm, I don't know, that sounds like it should be true for burnable blocks like planks too, which can be lit by flint and steel. I think for hasLitState it is more clear that it only means things like campfires.

Reply:
It doesn't just check if it has a lit state, it also checks if the campfire is not waterlogged or already lit

Reply:
Yes, but the effective purpose is to modify the Properties.LIT property.

Reply:
I was replying to `hasLitState`. That doesn't accurately describe what it does. `canBeLit` is ok.

Reply:
canBeLit is okay yes